tests/impl.cpp

```diff
@@ -8020,7 +8020,85 @@ result_t test_mm_round_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_round_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const float *_a = impl.mTestFloatPointer1;
+    const float *_b = impl.mTestFloatPointer1;
+    float f[4];
+    __m128 ret;
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 b = do_mm_load_ps(_b);
+    switch (i & 0x7) {
+    case 0:
+        f[0] = bankersRounding(_b[0]);
+        f[1] = _a[1];
+        f[2] = _a[2];
+        f[3] = _a[3];
+
+        ret = _mm_round_ss(a, b, _MM_FROUND_TO_NEAREST_INT | _MM_FROUND_NO_EXC);
+        break;
+    case 1:
+        f[0] = floorf(_b[0]);
+        f[1] = _a[1];
+        f[2] = _a[2];
+        f[3] = _a[3];
+
+        ret = _mm_round_ss(a, b, _MM_FROUND_TO_NEG_INF | _MM_FROUND_NO_EXC);
+        break;
+    case 2:
+        f[0] = ceilf(_b[0]);
+        f[1] = _a[1];
+        f[2] = _a[2];
+        f[3] = _a[3];
+
+        ret = _mm_round_ss(a, b, _MM_FROUND_TO_POS_INF | _MM_FROUND_NO_EXC);
+        break;
+    case 3:
+        f[0] = (int) _b[0];
```

Comment:
Replace with `f[0] = _b[0] > 0 ? floorf(_b[0]) : ceilf(_b[0]);` to avoid data lost.